src/pages/User/components/UserHeader/utils/handleAvatarUpload.ts

```diff
@@ -15,10 +19,49 @@ export const handleAvatarUpload = async (file: File | null | undefined) => {
       .catch((err) => console.error(err));
 
     const user = LocalStorage.getItem('user') as User;
-    LocalStorage.clear();
+    LocalStorage.remove('user');
     user.avatar_url = uploadImage.data.fileUrl;
     LocalStorage.set('user', user);
```

Comment:
I know that I suggested to remove 'user' hehe, but now that I think it, what about just setting the 'user' instead of remove it and add it again?

I think that with the LocalStorage.set('user', user) is enough, please check